schemas.py

```diff
@@ -3,17 +3,18 @@
 
 # Manager
 class ManagerBase(BaseModel):
-    manager_name: str
+    manager_firstname: str
+    manager_lastname: str
 
 class ManagerCreate(ManagerBase):
     password: str # When creating a manager it takes the actual password to hash and then store
-
-class ManagerDelete(ManagerBase):
-    pass
+    is_admin: bool
```

Comment:
Because `is_admin` is present in both `ManagerCreate` and `Manager` it should be in `ManagerBase`, which is where we put common attributes